integration-testing/election-manager/cypress/integration/tallies.ts

```diff
@@ -0,0 +1,33 @@
+import { electionMultiPartyPrimaryWithDataFiles } from '@votingworks/fixtures'
+
+describe('Election Manager can create SEMS tallies', () => {
+  it('Election Manager can tally results properly', () => {
+    cy.visit('/')
+    cy.get('input[type="file"]').attachFile(
+      'electionMultiPartyPrimarySample.json'
+    )
+    cy.contains('Election loading')
+    // cy.contains('Election Hash: abdfbe6a58')
+    cy.contains('Tally').click()
+    cy.contains('Import CVR Files').click()
+    cy.get('input[data-testid="manual-input"]').attachFile(
+      'multiPartyPrimaryCVRResults.jsonl'
+    )
+    cy.get('[data-testid="total-ballot-count"]').within(() =>
+      cy.contains('4,530')
+    )
+    cy.contains('Save Results File').click()
+    cy.get('[data-testid="manual-export"]').click()
+    cy.contains('Results Saved')
+    cy.task('readMostRecentFile', 'cypress/downloads').then((fileContent) => {
```

Comment:
Does cypress recommend not using async/await?

Reply:
yeah I believe you can't use async/await as cypress has its own similar task queue. https://docs.cypress.io/guides/core-concepts/introduction-to-cypress#Commands-Are-Asynchronous